Microsoft.RestApi.RestSplitter/RestSplitter.cs

```diff
@@ -256,6 +256,8 @@ private static void WriteToc(string sourceRootDir, string targetRootDir, OrgsMap
                 }
             }
 
+            TocConverter.GenerateOverviewNode(targetTocPath);
```

Comment:
Add one comment at here. We'd better to refine this TOC generation logic later to modify this during TOC generation.